package.json

```diff
@@ -5,34 +5,34 @@
   "version": "1.13.0",
   "author": "Open Source Team at New Relic <opensource@newrelic.com>",
   "dependencies": {
-    "@emotion/core": "^10.1.1",
-    "@emotion/styled": "^10.0.27",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.3.0",
     "@mdx-js/mdx": "^1.6.22",
     "@mdx-js/react": "^1.6.22",
-    "@newrelic/gatsby-theme-newrelic": "^1.38.0",
+    "@newrelic/gatsby-theme-newrelic": "^2.0.0",
     "@splitsoftware/splitio-react": "^1.2.4",
     "date-fns": "^2.17.0",
     "feather-icons": "^4.28.0",
-    "gatsby": "^2.32.8",
+    "gatsby": "^3.4.0-next.3",
```

Comment:
Looks good! Once we update this to use same version as theme (3.3.0) should be good to go

Reply:
Done!